tests/rptest/tests/random_node_operations_test.py

```diff
@@ -328,22 +336,24 @@ def maybe_enable_iceberg_for_topic(self, topic_spec: TopicSpec,
             client.alter_topic_config(topic_spec.name,
                                       TopicSpec.PROPERTY_ICEBERG_MODE,
                                       "key_value")
+            client.alter_topic_config(topic_spec.name,
```

Comment:
I believe you want `__REDPANDA_TEST_DISABLE_BOUNDED_PROPERTY_CHECKS` set if using this. Currently it seems to be set only for "with_chunked_compaction" test.

Reply:
10s is within the bounds though?